src/util/parseMarkdown.tsx

```diff
@@ -67,11 +65,13 @@ function tokenToReactNode(token: MarkedToken, key: string): React.ReactNode {
 const Markdown = styled(View)(theme => ({
   flexDirection: 'column',
   alignItems: 'stretch',
-  justifyContent: 'flex-start'
+  justifyContent: 'flex-start',
+  margin: theme.rem(0.5)
 }))
 
 const Paragraph = styled(View)(theme => ({
-  // backgroundColor: 'pink',
```

Comment:
Remove commented styles

Reply:
Was going to leave them there for debugging. Ye or ne?